docs/reference.md

```diff
@@ -263,9 +263,15 @@ As multiple users can reply to a form sent to a room, the activity waiting for a
 times (it can be seen as a sub execution of the current workflow). This also means that every activity defined after
 this activity or with a `activity-completed` event on this activity will run within its own sub execution.
 
+A form can be dedicated to be replied only once. In this case, `unique` is set to `true`.
```

Comment:
In 1-1 conversations or if the form has to be replied only once, the unique attribute on the form-replied event can be set to true. In that case the activity in the main execution of the current workflow and flow controls can be used as normal.